index.bs

```diff
@@ -169,6 +169,12 @@ interface VRDisplay : EventTarget {
    * created without preserveDrawingBuffer set to true will be cleared.
    */
   void submitFrame();
+
+  attribute EventHandler onactivate;
+  attribute EventHandler ondeactivate;
+  attribute EventHandler onblur;
+  attribute EventHandler onfocus;
+  attribute EventHandler onpresentchange;
```

Comment:
This was modeled after [`fullscreenchange`](https://developer.mozilla.org/en-US/docs/Web/Events/fullscreenchange) and [`pointerlockchange'](https://developer.mozilla.org/en-US/docs/Web/Events/pointerlockchange), which seemed to me like similar state transitions. I also see that the Page Visibility API include `visibilitychange`, so I'd say this verbiage is consistent with the rest of the web platform.